console/console-init/ui/src/modules/iot-device/components/DeviceFilter/DeviceFilter.tsx

```diff
@@ -163,15 +163,13 @@ const DeviceFilter: React.FunctionComponent<IDeviceFilterProps> = ({
       </SplitItem>
       <SplitItem>&nbsp;</SplitItem>
       <SplitItem>
-        <Dropdown
+        <DropdownWithKebabToggle
           id="filter-kebab-dropdown"
-          position={DropdownPosition.left}
-          isPlain
+          isPlain={true}
+          toggleId="device-filter-kebab"
```

Comment:
As we agreed on meeting, please stop using meaningless ID's if you hit some ID like this fix it, otherwise it's not testable.

Reply:
Done! We have actually two components here, one is the `dropdown` component and another is the `kebab toggle`.